src/config.ts

```diff
@@ -350,6 +350,7 @@ export const GAS_SCHEDULE_V0: FeeSchedule = {
     [ERC20BridgeSource.Bancor]: () => 4.5e5,
     [ERC20BridgeSource.MStable]: () => 8.5e5,
     [ERC20BridgeSource.Mooniswap]: () => 3.5e5,
+    [ERC20BridgeSource.MultiHop]: () => 0,
```

Comment:
This code path will get called even if `MultiHop` samples nothing now (ie, it's disabled), so we need this stub.